package.json

```diff
@@ -16,25 +16,19 @@
     "react-scripts": "^5.0.1",
     "react-websitecarbon-badge": "^1.0.5",
     "sass": "^1.53.0",
-    "typescript": "^4.7.4",
     "web-vitals": "^1.1.2"
   },
   "scripts": {
-    "start": "npm run generate-types && npm run release-date && react-scripts start",
+    "start": "npm run generate-types && npm run release-date && CI=false react-scripts start",
```

Comment:
Just realized I added `CI=false` to the `start` script and not the `build` script --this should work once I move it to the right place.